spec/requests/errors_spec.rb

```diff
@@ -34,6 +34,15 @@
 
   describe "/500" do
     it_behaves_like "an error response", 500, :internal_server_error
+
+    it "copes if warden is unavailable" do
+      allow_any_instance_of(ErrorsController) # rubocop:disable RSpec/AnyInstance
```

Comment:
Somewhat out-of-scope: could we [disable this Cop for controllers in general](https://github.com/rspec/rspec-rails/issues/1008#issuecomment-44233837)?

Reply:
I'm not quite sure what disabling it for all controllers would look like? I thought we mostly had the ability to ignore based on filenames rather than the granularity of an object type.

I'd expect we shouldn't need it frequently though. I feel like I'm in a really niche scenario here where it's rather impractical to test through conventional means. Whereas normally when you might want to stub a controller you'd be writing a view test where you have the ability to use `allow(view).to receive(:method)` to simulate a controller helpers.

Reply:
Sure, fair enough. This is one of the non-auto-correctable Cops that make adopting our RSpec styleguide hard, so I was just wondering if we could make it any easier.

By the way, I was thinking:

```
RSpec/AnyInstance:
  Exclude:
    - 'spec/requests/*'
```

Reply:
Cool, thanks. Yeah I'd hope we don't need to pull that out for controllers since it'd cover anything but it's nice to know we have it at our disposal.